modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language_es_AR.properties

```diff
@@ -288,11 +288,11 @@ action.ADD_DEPOT_ENTRY=Añadir repositorio
 action.ADD_DISCUSSION=Añadir un comentario
 action.ADD_DISPATCH_TRIGGER=Añadir desencadenador de envío
 action.ADD_DISPLAY_STYLE=Añadir estilo de presentación
-action.ADD_DOCUMENT=Añadir un documento
+action.ADD_DOCUMENT=Añadir Documento
 action.ADD_DOCUMENT_TYPE=Añadir tipo de documento
 action.ADD_ENTRY=Añadir una entrada
 action.ADD_EVENT=Añadir un evento
-action.ADD_EXPANDO=Añadir un campo personalizado
+action.ADD_EXPANDO=Añadir campo personalizado
```

Comment:
did you mean to edit these Language*.properties files?

Reply:
I didn't, these changes are from some commits that were also pushed because my branch was ahead of yours. This change, specifically is from LPS-77699 Update Translations commit.